spec/must_support_spec.rb

```diff
@@ -1,5 +1,6 @@
 RSpec.describe CancerPathologyDataSharingTestKit::MustSupportTest do # rubocop:disable RSpec/SpecFilePathFormat
-  let(:suite) { Inferno::Repositories::TestSuites.new.find('cpds_report_generation') }
+  let(:suite_id) { 'cpds_report_generation' }
+  let(:suite) { Inferno::Repositories::TestSuites.new.find(suite_id) }
   let(:session_data_repo) { Inferno::Repositories::SessionData.new }
```

Comment:
The lets defined in the runnable context can all be removed: https://github.com/inferno-framework/inferno-core/blob/main/spec/runnable_context.rb#L2-L5

I expect the test runner isn't needed anymore either.